modules/site-navigation/data/endpoints/recent.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Elementor\Modules\SiteNavigation\Data\Endpoints;
+
+use Elementor\Core\Base\Document;
+use Elementor\Core\Kits\Documents\Kit;
+use Elementor\Data\V2\Base\Endpoint;
+use Elementor\Plugin;
+use Elementor\TemplateLibrary\Source_Local;
+use Elementor\Utils;
+use WP_REST_Server;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Recent extends Endpoint {
+
+
+	public function register_items_route( $methods = WP_REST_Server::READABLE, $args = [] ) {
+		$args = [
+			'posts_per_page' => [
+				'description' => 'Number of posts to return',
+				'type' => 'integer',
+				'required' => true,
```

Comment:
Why not set default and set required to false?

Reply:
There is no logical default so I think its redundant. User should just pass the argument.